eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/common/operations/OperationsTestExecutor.java

```diff
@@ -279,7 +287,10 @@ private void processOperation(
                 testDefinition,
                 dataFileName,
                 testDefinition.getSpec().getGenesisSchemaDefinitions().getAttestationSchema());
+        var preState = state.commitChanges();
```

Comment:
reason we're using var?

Reply:
no reason, I forgot it. I use it when I want to go fast.